src/Transformer/TimestampTransformer.php

```diff
@@ -0,0 +1,30 @@
+<?php
+declare(strict_types=1);
+
+namespace Diaclone\Transformer;
+
+use Carbon\Carbon;
+use DateTime;
+use Diaclone\Resource\ResourceInterface;
+
+class TimestampTransformer extends AbstractTransformer
+{
+    public function transform(ResourceInterface $resource)
+    {
+        /** @var DateTime $value */
+        $value = $this->getPropertyValueFromResource($resource);
+
+        if (is_string($value)) {
+           $value = new DateTime($value);
+        }
+
+        return $value->getTimestamp();
+    }
+
+    public function untransform(ResourceInterface $resource)
+    {
+        $value = $resource->getData();
+
+        return $value ? (new DateTime())->setTimestamp((int) $value) : null;
```

Comment:
are you avoiding the use of `Carbon` on purpose? it's included in the composer.json file so it might be easier to use `Carbon::createFromTimestamp((int)$value)`